nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/test/java/org/nuxeo/elasticsearch/test/rest/RestESDocumentsTest.java

```diff
@@ -128,7 +128,7 @@ public void iCanPerformESQLPageProviderOnRepository() throws IOException, Interr
 
         JsonNode node = mapper.readTree(response.getEntityInputStream());
         // Verify results
-        assertEquals(19, getLogEntries(node).size());
+        assertEquals(20, getLogEntries(node).size());
```

Comment:
weird that if we are not indexing the root document anymore we have one more entry ?